albam/engines/mtfw/structs/lmt-67.ksy

```diff
@@ -0,0 +1,66 @@
+meta:
+  endian: le
+  file-extension: lmt
+  id: lmt
```

Comment:
Beware that using the same id as in lmt.ksy overwites the generated parser. Does Re5 lmt import still works?
Also, Is it possible to consolidate both ksy into one file?
That should be preferrable unless the structure varies a lot and requires way too many "ifs"